cloud_cleaner/bin/entrypoint.py

```diff
@@ -26,9 +26,13 @@ def cloud_clean(args=sys.argv[1:],  # pylint: disable=W0102
     config.parse_args()
     # Call the process method for the target resource type
     print("Options parsed, fetching resources")
-    ALL_RESOURCES[config.get_resource()].process()
     if config.get_arg("force"):
+        ALL_RESOURCES[config.get_resource()].prep_deletion()
+        ALL_RESOURCES[config.get_resource()].process()
         print("Resources fetched, cleaning")
         ALL_RESOURCES[config.get_resource()].clean()
     else:
         print("No changes made, force option not enabled")
+    ALL_RESOURCES[config.get_resource()].process()
```

Comment:
Before I merge this, I would just like a little confirmation on this line: Is this going to be called twice (here and on L31) in the case of a call with `force == True`? It's not ideal, but I get it.

Reply:
Yes it is going to be called twice in that case. I agree it isn't ideal, but I felt like it could still make sense. One process call before the actually deletion, so we know what to delete/clean(), and one after to know which servers to email or not(as well as to include a process call regardless of whether force == True, something in the current version of cloud cleaner).
I could move this process call to inside of the clean() function of server.py instead(at the end), which would keep functionality but could make the logic of entrypoint.py nicer if you would prefer that.